README.md

```diff
@@ -1,28 +1,33 @@
-![Spectral logo](./docs/img/spectral-banner.png)
-
-[![CircleCI](https://img.shields.io/circleci/build/github/stoplightio/spectral/master)](https://circleci.com/gh/stoplightio/spectral) [![NPM Downloads](https://img.shields.io/npm/dw/@stoplight/spectral?color=blue)](https://www.npmjs.com/package/@stoplight/spectral) [![Treeware (Trees)](https://img.shields.io/treeware/trees/stoplightio/spectral)](https://plant.treeware.earth/stoplightio/spectral)
-
-A flexible JSON/YAML linter, with out of the box support for OpenAPI v2/v3 and AsyncAPI v2.
+<div align="center" width="1000">
+  <a href="https://stoplight.io/api-mocking?utm_source=github&utm_medium=spectral&utm_campaign=readme">
```

Comment:
Could we use markdown instead of HTML?
You can leave HTML for the `div` above (cause of `align`) and perhaps the image right below.

Reply:
oh, the URL is not correct. I suppose you don't want to point at mocking, do you?

Reply:
Thanks! Fixed this.